packages/dashboard/src/components/viewStyleBar/stories/index.js

```diff
@@ -29,13 +28,23 @@ import ViewStyleBar from '..';
 export default {
   title: 'Dashboard/Components/ViewStyleBar',
   component: ViewStyleBar,
+  // TODO: fix select dropdown
+  argTypes: {
+    layoutStyle: {
+      options: Object.values(VIEW_STYLE),
+    },
+  },
```

Comment:
I haven't been able to get the dropdown `select` control to work. Here it is using `radio` button. I read that by default Storybook will use `radio` if there are less than 5 options. (We only have two options here). So this is fine as a radio select. Regardless `argTypes` still are not working correctly. Could be because my brain is mush at this point, or because I am mixing and matching versions of Storybook and add-on/controls. [See comment above](https://github.com/google/web-stories-wp/pull/10274/files#r788255757)

Reply:
Perhaps this works?

```js
argTypes: {
    layoutStyle: {
      options: Object.values(VIEW_STYLE),
      control: {
        type: 'select',
      }
    },
  },
```

Reply:
Yeah, I've tried that and a few other suggestions from the docs. Unfortunately it seems none of the `argtypes` configurations work properly. Once I get all the knobs converted to controls, I can remove the `knobs` add-on package and properly upgrade Storybook. I _hope_ that will help. Though it doesn't seem likely. But I will dig into this a bit more and check our configuration setup for Storybook to see if there is anything there that may be leading to Storybook getting confused.

Reply:
@BrittanyIRL and I got it working properly. It was due to the docs add-on. This is where the autogenerated `argTypes` derive, and since I hadn't yet removed the  `@storybook/addon-docs` (or any of the other essential addons) it was not getting the updated `argTypes`. Now that it is all working as expected I can continue to update the stories to remove `knobs` and move `actions` into the `argTypes`